src/frontend/Komponenter/Behandling/Høyremeny/Høyremeny.tsx

```diff
@@ -57,7 +59,18 @@ export enum Høyremenyvalg {
 
 const Høyremeny: React.FC<IHøyremenyProps> = ({ behandlingId, åpenHøyremeny }) => {
     const [aktivtValg, settAktivtvalg] = useState<Høyremenyvalg>(Høyremenyvalg.Logg);
-    const { settÅpenHøyremeny } = useBehandling();
+    const { settÅpenHøyremeny, behandling } = useBehandling();
+
+    useEffect(() => {
+        if (
+            behandling.status === RessursStatus.SUKSESS &&
+            [BehandlingStatus.UTREDES, BehandlingStatus.FATTER_VEDTAK].includes(
```

Comment:
Kanskje trekke ut den ene sjekken til en egen funksjon for bedre lesbarhet:

const behandlingErUnderArbeid = (behandlingStatus: BehandlingStatus) => {
return [BehandlingStatus.UTREDES, BehandlingStatus.FATTER_VEDTAK].includes(behandling.data.status)
}

Reply:
I fix